<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>615</width>
        <height>362</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>248</red>
            <green>248</green>
            <blue>248</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>255</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>248</red>
            <green>248</green>
            <blue>248</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Base">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>248</red>
            <green>248</green>
            <blue>248</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Window">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>248</red>
            <green>248</green>
            <blue>248</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="profileTitleLayout">
         <item>
          <widget class="QLabel" name="profileIconLabel">
           <property name="maximumSize">
            <size>
             <width>18</width>
             <height>18</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="res.qrc">:/main/filter_none-24px.svg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="profileName">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="placeholderText">
            <string>Profile Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="profileActive">
           <property name="text">
            <string>Active</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="exportButton">
           <property name="text">
            <string>Export Profile</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/main/description-24px.svg</normaloff>:/main/description-24px.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="processExpandBtn">
         <property name="styleSheet">
          <string notr="true">background: rgba(100, 100, 100,0); border: 1px solid; border-radius: 3px; border-color: rgb(146, 146, 146); margin: 1px;</string>
         </property>
         <property name="text">
          <string>Executables</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/main/keyboard_arrow_down-24px.svg</normaloff>
           <normalon>:/main/keyboard_arrow_up-24px.svg</normalon>:/main/keyboard_arrow_down-24px.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="processBox" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Define application processes to watch for their creation(start) or deletion(stop).</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="processAddBtn">
         <property name="text">
          <string>Add Executable to watch</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/main/add-24px.svg</normaloff>:/main/add-24px.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="taskExpandBtn">
         <property name="styleSheet">
          <string notr="true">background: rgba(100, 100, 100,0); border: 1px solid; border-radius: 3px; border-color: rgb(146, 146, 146); margin: 1px;</string>
         </property>
         <property name="text">
          <string>Tasks</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/main/keyboard_arrow_down-24px.svg</normaloff>
           <normalon>:/main/keyboard_arrow_up-24px.svg</normalon>:/main/keyboard_arrow_down-24px.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="taskBox" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="tasksLabel">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Define tasks to execute once &lt;span style=&quot; font-weight:600;&quot;&gt;any&lt;/span&gt; of the above defined processes have been created or deleted. Add conditions per task eg.: start VoiceAttack only if SteamVR is running&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="taskAddBtn">
         <property name="text">
          <string>Add Task</string>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/main/add-24px.svg</normaloff>:/main/add-24px.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
